src/smk/tool/geomark/tool-geomark.js

```diff
@@ -0,0 +1,49 @@
+include.module( 'tool-geomark', [ 
+    'tool.tool-base-js',
+    'tool.tool-widget-js',
+    'tool.tool-panel-js',
+    'tool-geomark.panel-geomark-html'
+], function ( inc ) {
+    "use strict";
+
+    Vue.component( 'geomark-widget', {
+        extends: SMK.COMPONENT.ToolWidgetBase,
+    } )
+
+    Vue.component( 'geomark-panel', {
+        extends: SMK.COMPONENT.ToolPanelBase,
+        template: inc[ 'tool-geomark.panel-geomark-html' ]
+    } )
+    // _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
+    //
+    return SMK.TYPE.Tool.define( 'GeomarkTool',
+        function () {
+            SMK.TYPE.ToolWidget.call( this, 'geomark-widget' )
+            SMK.TYPE.ToolPanel.call( this, 'geomark-panel' )
+        },
+        function ( smk ) {
+            var self = this
+
+            this.changedActive( function () {
+                if ( self.active ) {
+                    smk.$viewer.map.pm.enableDraw('Polygon', {
+                        continueDrawing: true
+                    });
+                }
+                else {
+                    smk.$viewer.map.pm.disableDraw();
+                }
+            } )
+
+            var editableLayers = new L.FeatureGroup();
+            smk.$viewer.map.addLayer(editableLayers);
+
+            smk.$viewer.map.on('pm:create', function(e) {
+                var type = e.layerType,
+                    layer = e.layer;
```

Comment:
For the sake of readability, could you please declare each variable individually?
ie.
var type = e.layerType;
var layer = e.layer;

Reply:
Done